aten/src/ATen/native/mps/operations/Convolution.mm

```diff
@@ -372,7 +372,6 @@ static Tensor mps_convolution_backward_input(IntArrayRef input_size,
   using namespace at::native::mps;
   using namespace mps;
   bool is3DConv = grad_output_t.dim() == 5;
-
   if (!is_macos_13_or_newer(MacOSVersion::MACOS_VER_15_1_PLUS)) {
```

Comment:
```suggestion
    Tensor grad_output_contiguous = is_macos_13_or_newer(MacOSVersion::MACOS_VER_15_0_PLUS) ?
                                  grad_output_t : grad_output_t.contiguous();
```

Then update to use grad_output_contiguous everywhere else in the code

This was why I was copying your code from your PR to mine, just wanted to keep your implementation at the same time try having a fallback for anything lower than macOS15. I am really sorry that straight away caused an error. I wanted the PR to get merged into the branch you are working, so when I did changed that it automatically caused a lot of issues.

Reply:
Why do you think this is needed? I.e. looks at the test sanity? On both MacOS-13, 14 and 15, results of the backward path are now identical, so existing logic (in `Placeholder` constructor) works, because it's no longer told to do something weird in case of channels last. And as result, on MacOS-15 no copies will be needed for channels_last case, but on MacOS-14 there will be copies, but results will be correct

Reply:
Oh I was under the impression it was not working on older Mac devices 
(https://github.com/pytorch/pytorch/issues/142344#issuecomment-2541962490). I do not have an older Mac to test this though, but again thank you for addressing this issue, and sorry for confusion.